spec/features/review_card_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+require 'support/review_translation_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.